setup.py

```diff
@@ -158,6 +158,10 @@ def readme():
         'crefl_sharpen_awips=polar2grid.compositors.rgb:CreflRGBSharpenCompositor',
         ],
     'polar2grid.frontend_class': [
+        'viirs_edr_flood=polar2grid.readers.mersi2_l1b:Frontend',
+        'viirs_edr_active_fires=polar2grid.readers.virr_l1b:Frontend',
```

Comment:
Copy/paste error, wrong reader. Same for the frontend arguments below for a couple of them.